website/server/controllers/api-v3/groups.js

```diff
@@ -1086,4 +1086,100 @@ api.inviteToGroup = {
   },
 };
 
+/**
+ * @api {post} /api/v3/groups/:groupId/add-manager Add a manager to a group
+ * @apiName AddGroupManager
+ * @apiGroup Group
+ *
+ * @apiParam (Path) {UUID} groupId The group _id ('party' for the user party and 'habitrpg' for tavern are accepted)
+ *
+ * @apiParamExample {String} party:
+ *     /api/v3/groups/party/add-manager
+ *
+ * @apiBody (Body) {UUID} managerId TThe user _id of the member to promote to manager
+ *
+ * @apiSuccess {Object} data An empty object
+ *
+ * @apiError (400) {NotAuthorized} managerId req.body.managerId is required
+ * @apiUse groupIdRequired
+ */
+api.addGroupManager = {
+  method: 'POST',
+  url: '/groups/:groupId/add-manager',
+  middlewares: [authWithHeaders()],
+  async handler (req, res) {
+    let user = res.locals.user;
+    let managerId = req.body.managerId;
+
+    req.checkParams('groupId', res.t('groupIdRequired')).notEmpty(); // .isUUID(); can't be used because it would block 'habitrpg' or 'party'
```

Comment:
since these errors are not meant for end users, can we use the new `apiMessages` lib like here https://github.com/HabitRPG/habitica/commit/939712ad1f8e8c8e228907b30844fcd3bb5dae51#diff-b950a7c557b35ea58bc2685532c46170R285 So the i18n strings to load on the client don't grow in size. It should be used for all API messages meant for devs that don't need to be translated